gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -367,19 +384,19 @@ protected String createAndShowDialog(String current) {
             new GridData(GridData.FILL_HORIZONTAL));
         createLabel(durGroup, "Start at:");
         startType = Widgets.createDropDown(durGroup);
-        startType.setItems("Beginning", "Manual", "Frame");
+        startType.setItems(Arrays.stream(StartType.values()).map(Enum::name).toArray(String[]::new));
```

Comment:
This isn't using the map you created above, so if you choose to not use the name, then this needs to be fixed to whatever you choose.